packages/files-ui/src/Contexts/FilesContext.tsx

```diff
@@ -662,7 +662,7 @@ const FilesProvider = ({ children }: FilesContextProps) => {
       return Promise.resolve()
     } catch (error: any) {
       console.error(error)
-      let errorMessage = `${t`An error occurred: `} ${typeof(error) === "string" ? error : error.error.message ? error.error.message : ""}`
+      let errorMessage = `${t`An error occurred: `} ${error || error.message || error.error.message || ""}
```

Comment:
suggestion gone wrong. BTW this should be tested, I did not test it :D
```suggestion
      let errorMessage = `${t`An error occurred: `} ${error || error.message || error.error.message || ""}`
```

Reply:
Wouldn't this way create problems if `error` was an object and not a string ?

Reply:
My preference here would be to just use `let errorMessage = `${t`An error occurred: `} ${typeof(error) === "string" ? error : error?.error?.message ? error.error.message : ""}``

Reply:
We can leave the last one at least
```suggestion
      let errorMessage = `${t`An error occurred: `} ${typeof(error) === "string" ? error : error?.error?.message || ""}`
```

Reply:
Oh lord 😂